src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -57,10 +59,15 @@ object Run  {
   }
 }
 
-case class Run(name: String, pipeline: Pipeline, tag: String) {
+case class Run(jesId: String, pipeline: Pipeline, tag: String) {
+
+  val dataAccess = DataAccess()
```

Comment:
Yes, IMO there should be a single `object DataAccessObject` instantiating one `DataAccess` once and for all that we could use everywhere.